ipyvolume/widgets.py

```diff
@@ -51,6 +51,7 @@ class Mesh(widgets.DOMWidget):
 #                                     default_value="green").tag(sync=True)
 #    geo = traitlets.Unicode('diamond').tag(sync=True)
     wire = traitlets.CBool(default_value=False).tag(sync=True)
+    visible = traitlets.CBool(default_value=True).tag(sync=True)
```

Comment:
what about visible/visible_faces/visible_lines (instead of the 'wire', which is an odd name)? So the faces would be visible when `visible && visible_faces` is true, and the lines would be when `visible & visible_lines` is true. (Note that I prefer visible_xxx for tab completion, instead of xxx_visible).